lib/pbench/server/api/resources/__init__.py

```diff
@@ -391,6 +391,8 @@ def convert_username(value: Union[str, None], _) -> Union[str, None]:
         ) from e
 
     if not user:
+        # TODO: Should we change the status to FORBIDDEN as we dont want other
+        # users to know about the usernames in our db
         raise ConversionError(value, "username", http_status=HTTPStatus.NOT_FOUND)
 
     return str(user.id)
```

Comment:
Well, this function `convert_username` is called when the API parameter is of type `USER` (line 648 of this file) so it's very much in use. I am not changing the return type from user.id to User in this PR as I will also have fix the corresponding APIs that use `USER` param type.

Reply:
My concern about returning `user.id` has been allayed by your change to make `user.id` be the OIDC UUID instead of the `Users` table row ID, so that's all good, now. 

My remaining concerns here are the text of that comment.  First, if it's meant to express a question, it would be good if it ended with a question mark (`?`).  Aside from that, as I said above, I don't think we need to be concerned about exposing usernames, here:  if a user asks for a username which doesn't exist, I think that reporting "not found" is fine.